### ISSUE TITLE
Patches required for next release

### ISSUE THREAD

**Issue by drojf** [07th-mod/umineko-answer#17]
If anything needs to be done before the next release, please document them in this thread.

drojf - please use the latest script file in the next release (done automatically?)

**Comment by drojf** [comment]
Missing sprites:

Due  to the script changes, some inserted lines don't have sprites. I decided to extract all the sprite file paths from the umitweak release to ensure we have all the sprites even if I failed to insert a umitweak sprite. 

Long story short, please update the sprite pack with the following 24 images:
[missing_sprites_v.1.zip](https://github.com/07th-mod/umineko-answer/files/1605486/missing_sprites_v.1.zip)

**Comment by drojf** [comment]
@irlPM  figured out the  bmp/background/efe folder was missing as he had deleted it by accident. This should be included in the next release:

## ITALO I MADE A MISTAKE, I USED THE WRONG FOLDER STRUCTURE. 

IT SHOULD BE `bmp\background\efe`, I added an 's' to 'background' by accident....

https://github.com/07th-mod/resources/releases/download/Bern/Missing.bmp.background.efe.05-01-18.zip

## ITALO I MADE A MISTAKE, I USED THE WRONG FOLDER STRUCTURE.

**Comment by drojf** [comment]
In the above release, it looks like the missing_images.zip was not merged into the file (or perhaps the other way round - it overwrote the missing_images images which were in the orginial graphis zip).

The missing_images.zip should be applied, it can be found here:
https://mega.nz/#!xGollRRR!F8mVSaVIouEbPAu6MFq-QAEkh9LE8sehKcj1cTTQtLo

**Comment by drojf** [comment]
In the above release, it looks like the missing_images.zip was not merged into the file (or perhaps the other way round - it overwrote the missing_images images which were in the orginial graphis zip).

The missing_images.zip should be applied, it can be found here:
https://mega.nz/#!xGollRRR!F8mVSaVIouEbPAu6MFq-QAEkh9LE8sehKcj1cTTQtLo

**Comment by irlPM** [comment]
Some kagera were missing and cuts too, here is the download link with the right structure: https://mega.nz/#!CpVFlZ4B!vptM6X41f2vVMcypx3VIJ3d94fYSJqfrrRWnGqTTXXU

**Comment by drojf** [comment]
I have reverted back to using a script animation for the ura8 sequence. Due to this, some additional assets are needed. For now I've put them in the 'movie' folder, but it can be anywhere, I just need to update the script.

INSTRUCTIONS:
- copy contents of the movie folder to the movie folder in the root directory of the game
- ignore the attached script file 

[google drive ura8 link](https://drive.google.com/file/d/1mXieZcRVwC6pgub866vmhWim0h5czTov/view?usp=sharing)

**Comment by danpmss** [comment]
I hope this isn't the wrong place to post.

I'm just here to let you know that Manga Gamer made a translation patch for Chiru not so long ago fixing many typos across the whole thing.

I hope this doesn't affect your progress, I just wanted to be sure to let you know if this is something worth noticing at all (I don't even know how these things works in the patch).

**Comment by ItaloKnox** [comment]
@danpmss as far as I know, we merged this patch already.

**Comment by danpmss** [comment]
Oh, that's great to know. I'll get out of the way now 👍 
Keep the great job, guys.

**Comment by drojf** [comment]
@danpmss 
>fixing many typos across the whole thing.

Unless I made a mistake, there were very few actual fixes, you can see them here: https://github.com/07th-mod/umineko-answer/commit/00a9300ddf45738cbc7a7075b4917f841724a82e

Was there _another_ update? afaik there was only one update since the game's release, but maybe they released another one recently; I haven't opened steam in a while.

**Comment by drojf** [comment]
The following image needs to REPLACE the existing image in the backgrounds `bg\` directory. Previously it was an image of rudolf by himself with no background - this image adds the background and forefround.
 
[mlib_1f_r.zip](https://github.com/07th-mod/umineko-answer/files/1617027/mlib_1f_r.zip)

**Comment by drojf** [comment]
For the **voice only** version of the script, the following video should be included in the release. Folder structure is included - the movie file is placed at `movie/ep8end.mpg`:

https://drive.google.com/file/d/1oiDQkpDVNhi8h5PW9qI7xKKOcpAijlEv/view?usp=sharing

**Comment by drojf** [comment]
For the **normal release**, Fort has updated the logos to be in english, and added some bullet effects which were not yet replaced. The folder structure is included. 

[Fort's images](https://drive.google.com/file/d/19dbZ6N9cuAcz81p9kCxXz8P8RwRK0COA/view?usp=sharing)

**Comment by drojf** [comment]
For the normal release, there were still some issues with the credits. Two new credit images are included in the zip file (should overwrite any existing images):

[Updated Credits 21-01-18 Gdrive](https://drive.google.com/file/d/1aZ7XSQNQbfBu-Xh14prgZjL1pj-s434w/view?usp=sharing)

**Comment by drojf** [comment]
# Umineko Answer Arcs Voice and Graphics Patch v1.0.0

# Umineko Answer Arcs Voice Patch v0.6.0 

Putting a marker here so we know where we're up to. All previous patches should have been included in the above release.

**Comment by drojf** [comment]
[texthotfix.zip](https://github.com/07th-mod/umineko-answer/files/1680651/texthotfix.zip)

This zip file contains fort's fixes for issue https://github.com/07th-mod/umineko-answer/issues/31

**Comment by drojf** [comment]
Another image was missing 'kannon_r.bmp' - folder structure is in the zip file:
[kannon_r_rclick_back_ep5.zip](https://github.com/07th-mod/umineko-answer/files/1689271/kannon_r_rclick_back_ep5.zip)

This is the fade image used when you exit back to the title menu.

**Comment by Forteissimo** [comment]
EP6 descriptions for Rosa and Maria which had the places wrong (Changed to parlor)

[ep6.zip](https://github.com/07th-mod/umineko-answer/files/1691369/ep6.zip)

**Comment by drojf** [comment]
The following images weren't been resized for the 'golden snow' scene. They should be located at `bmp\background\efe` - should overwrite the old images :

[cut_golden_snow_images.zip](https://github.com/07th-mod/umineko-answer/files/1691453/cut_golden_snow_images.zip)

- cut_w_efe_down.png
- cut_w_efe_up.png
- rub1,2,3,4 .png

**Comment by drojf** [comment]
# Umineko Answer Arcs Voice Patch v0.6.1

Putting a marker here so we know where we're up to. All previous patches should have been included in the above release.

**Comment by drojf** [comment]
Inexplicably, there are six voice files which aren't in our v0.6.0 release 'voice.zip' file, but exist in the ps3 game/umitweak's original release. I've attached them here, and checked they actually play in game (eg they are not corrupted):

[missing_voice_files_12-02-2018.zip](https://github.com/07th-mod/umineko-answer/files/1715907/missing_voice_files_12-02-2018.zip) (appropriate folder structure included)

```
voice\34\33300121.ogg
voice\34\33300121_1.ogg
voice\34\33300122.ogg
voice\34\33300123.ogg
voice\34\33300123_1.ogg
voice\34\33300124.ogg
```

**Comment by ItaloKnox** [comment]
I'll recompile the patch with them.

**Comment by ItaloKnox** [comment]
New build should be online. Already updated the meta and the installer to use the new build (should be in lzma now, saves around 200mb).

**Comment by drojf** [comment]
# ----------------------------------------------------

Small issue but one of gohda's sprite's arms was cut off slightly. I think this sprite is only used once in the game so not that important to replace it. (I manually made this sprite but must have messed up). To be added when we merge the update files into one. Here's the updated sprite (it goes in the `sprites\goh\1` folder)

[goh_a12_omakase1](https://user-images.githubusercontent.com/1249449/37250023-1e2bb384-2547-11e8-8198-fdbe32900fd8.png)
